test/test-network/systemd-networkd-tests.py

```diff
@@ -2450,6 +2450,73 @@ def test_ifb(self):
 
         self.wait_online(['ifb99:degraded'])
 
+    def test_rps_cpu_0(self):
+        copy_network_unit('12-dummy.netdev', '25-rps-cpu-0.link')
+        start_networkd()
+
+        self.wait_links('dummy98')
+
+        output = check_output('cat /sys/class/net/dummy98/queues/rx-0/rps_cpus')
+        print(output)
+        self.assertEqual(output, '1')
+
+    def test_rps_cpu_1(self):
+        cpu_count = os.cpu_count()
+        if cpu_count < 2:
+            return
```

Comment:
Instead, please use `@unittest.skipUnless(...)` or similar. Please see `test_macsec()` for example.